x-pack/plugins/security_solution/public/common/containers/related_entities/related_hosts/index.tsx

```diff
@@ -50,7 +50,8 @@ export const useUserRelatedHosts = ({
     errorMessage: FAIL_RELATED_HOSTS,
     abort: skip,
   });
-  const isNewRiskScoreModuleAvailable = useIsExperimentalFeatureEnabled('riskScoringRoutesEnabled');
```

Comment:
Nit: could we have just changed the definition here without changing the name? That might've helped with the # of changes needed here 🤷 

Edit: I see that in the hook itself you're using both "available" and "installed" as concepts. I'm still not sure that merits changing this variable name, but I understand the motivation.